images/hook/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/dumb-init /bin/sh
+set -e
+
+if [ $1 = "update" ]; then
+    echo "Starting update, changeset: $RIG_CHANGESET"
+    echo "Deleting old replication controller rc/log-forwarder"
+    rig delete rc/log-collector --resource-namespace=kube-system
+    echo "Creating or updating resources"
+    rig upsert -f /var/lib/gravity/resources/resources.yaml --debug
+    echo "Checking status"
+    rig status $RIG_CHANGESET --retry-attempts=120 --retry-period=1s --debug
+    echo "Freezing"
+    rig freeze
+else
```

Comment:
It does read weird: invoking the hook w/o arguments will rollback. I guess this is only useful with the rollback hook, since the rollback is not necessary during update in case of a failure (correct me if I'm wrong).